remacs-generated/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "remacs-generated"
+version = "0.1.0"
+description = "Remacs Generated"
```

Comment:
This description could be much better.

Reply:
@yyoncho mentioned that it would be good to drop the name remacs completely for political reasons. I think he's right, so maybe we should use something completely different.